lib/persisters/s3.js

```diff
@@ -41,6 +41,10 @@ class S3 {
 }
 
 module.exports = (opts) => {
+  console.log("OPTS:", opts)
```

Comment:
should prob remove all the `console` statements if this is going to be a published module. the module `debug` is great for handling output